stubs/haskell-http-client-tls/src/Main.hs

```diff
@@ -84,6 +86,6 @@ injectCA caBundle p =
 
 injectCiphers :: ClientParams -> ClientParams
 injectCiphers p =
-  p { clientSupported = supported { supportedCiphers = ciphersuite_all } }
+  p { clientSupported = supported { supportedCiphers = ciphersuite_strong } }
```

Comment:
Is `ciphersuite_default` also sufficient?  Or is the default "too permissive"?

Reply:
They result in the same. So, let's use `default`.

Reply:
My intent when using `ciphersuite_all` is to let the TLS server select the appropriate ciphersuite for a particular test. In TLS Handshake `ClientHello` advertises all supported ciphersuites and `ServerHello` selects the one to be used. Changing it to more restrictive list of ciphersuites might limit the testing and provide skewed results.

It's worth testing every supported ciphersuite to find the problems they might contain.

Reply:
Well some ciphersuites are implemented, but deprecated (not recommended) and should generally not be enabled. So the question comes down to what a particular test is intended to cover.  If it is a test of the correctness of the implementation of even deprecated ciphers, then it should "all", but if the test is to make sure that weak options are not enabled, then it should use "default" or "strong".  So decide what the goal is and choose wisely.

Reply:
`ciphersuite_all` will be removed someday. It should not be used anyway.